extensions/wikia/AdEngine/js/lookup/rubiconVulcan.js

```diff
@@ -0,0 +1,125 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.rubiconVulcan', [
+	'ext.wikia.adEngine.lookup.lookupFactory',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (factory, doc, log, win) {
+	'use strict';
+
+	var accountId = 7450,
+		config = {
+			oasis: {
+				INCONTENT_BOXAD_1: {
+					size: [480, 320],
+					siteId: 55412,
+					zoneId: 260296
+				}
+			},
+			mercury: {}
+		},
+		libraryUrl = '//ads.aws.rubiconproject.com/video/vulcan.min.js',
+		logGroup = 'ext.wikia.adEngine.lookup.rubiconVulcan',
+		priceMap = {},
+		sizeMap = {
+			'480x320': 101,
+			'640x480': 201
+		},
+		slots = {},
+		vulcanCpmKey = 'cpm',
+		vulcanUrlKey = 'depot_url';
+
+	function defineSingleSlot(slotName, slot) {
+		var size = slot.size[0] + 'x' + slot.size[1],
+			slotDefinition = {
+				'accountId': accountId,
+				'site_id': slot.siteId,
+				'size_id': sizeMap[size],
+				'zone_id': slot.zoneId,
+				'width': slot.size[0],
+				'height': slot.size[1],
+				'rand': Math.round(1000000000 * Math.random()),
+				'key1': 'value1'
+			};
+
+		log(['defineSlot', slotName, slotDefinition], 'debug', logGroup);
+		win.rubicontag.video.defineSlot(slotName, slotDefinition);
+	}
+
+	function defineSlots(onResponse) {
+		Object.keys(slots).forEach(function (slotName) {
+			defineSingleSlot(slotName, slots[slotName]);
+		});
+
+		win.rubicontag.video.run(onResponse);
+	}
+
+	function getSlotParams(slotName) {
+		// TODO: implement me
+		var parameters = {};
+
+		log(['getSlotParams', slotName, parameters], 'debug', logGroup);
+		return parameters;
+	}
+
+	function encodeParamsForTracking(params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "params". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)